README.adoc

```diff
@@ -51,55 +51,56 @@ In this section, you will clone a Spring Boot application and test it locally.
-./mvnw com.microsoft.azure:azure-spring-apps-maven-plugin:1.14.0:config -DadvancedOptions
+./mvnw com.microsoft.azure:azure-spring-apps-maven-plugin:1.16.0:config
 ----
 +
- - Command explanation:
-   * `export MAVEN_OPTS="--add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.io=ALL-UNNAMED"` is used to avoid the problem tracked in https://github.com/microsoft/azure-maven-plugins/issues/2222[azure-maven-plugins#2222].
-   * This maven goal will first authenticate with Azure, if you have logged in with https://docs.microsoft.com/en-us/cli/azure/[Azure CLI], it will consume its existing authentication token. Otherwise, it will get you logged in with https://github.com/microsoft/azure-maven-plugins/wiki/Azure-Maven-Plugin/[azure-maven-plugin] automatically.
-
- - Input necessary information:
-   * Select subscription: Select the subscription which host the Azure Spring Apps you just created.
-   * Select Azure Spring Apps for deployment: Select the Azure Spring Apps you just created.
-   * Input runtime Java version(Java 11): Java 17
-   * Expose public access for this app boot-for-azure? (y/N): y
-   * For other options, just select the default value.
- - Here is a sample terminal output:
+ - This maven goal will first authenticate with Azure, if you have logged in with https://docs.microsoft.com/en-us/cli/azure/[Azure CLI], it will consume its existing authentication token. Otherwise, it will get you logged in with https://github.com/microsoft/azure-maven-plugins/wiki/Azure-Maven-Plugin/[azure-maven-plugin] automatically.
+
+ - After login successfully, you will get the subscription list like this:
 +
 ----
-~@Azure:~/gs-spring-boot/complete$ ./mvnw com.microsoft.azure:azure-spring-apps-maven-plugin:1.14.0:config -DadvancedOptions
-[INFO] Scanning for projects...
-[INFO]
-[INFO] ------------------------------------------------------------------------
-[INFO] Building boot-for-azure 0.0.1-SNAPSHOT
-[INFO] ------------------------------------------------------------------------
-[INFO]
-[INFO] --- azure-spring-apps-maven-plugin:1.14.0:config (default-cli) @ demo ---
-...
 Available subscription:
- 1. xxx
-...
-Select subscription [1-105] (20):
-...
+1. Demo Subscription A (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx)*
+2. Demo Subscription B (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx)
+3. Demo Subscription C (xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx)
+... // Omit other subscriptions
+Select subscription [1-xx] (1):
+----
++
+IMPORTANT: If there is error like `Execution of class com.microsoft.aad.msal4j.AcquireTokenSilentSupplier failed.` and get the subscription list successfully. You can just ignore the error and continue the following steps. Here is more information about this error: https://github.com/microsoft/azure-maven-plugins/issues/2270[azure-maven-plugins#2270].
+
+ - Select the subscription which host the Azure Spring Apps you just created. Then you will get the Azure Spring Apps list:
```

Comment:
Change to:

Select the subscription that hosts the Azure Spring Apps you created. Then you see the Azure Spring Apps list: